### PR TITLE
Use unique param for affectedRows

### PR DESCRIPTION
**Pull request by loay**
connect to https://github.com/strongloop/loopback-connector-postgresql/issues/204

### CODE REVIEW

**Comment by loay** [comment]
@bajtos PTAL. Thanks

File: lib/postgresql.js [hunk 1]
```diff
@@ -153,7 +153,7 @@ PostgreSQL.prototype.executeSQL = function(sql, params, options, callback) {
         switch (data.command) {
           case 'DELETE':
           case 'UPDATE':
-            result = {count: data.rowCount};
+            result = {affectedRows: data.rowCount};
```
Comment (bajtos) [inline lib/postgresql.js]: Please preserve the `count` property for backwards compatibility.

```js
result = {affectedRows: data.rowCount, count: data.rowCount};
```

**Comment by bajtos** [comment]
Let's leave the tests out of scope of this patch, since they have to be implemented in a different repository anyways. However, I think it makes sense to add a postgresql-specific test to verify that the `count` property is preserved.

File: lib/postgresql.js (hunk 1 above)
Comment (loay) [inline lib/postgresql.js]: Done and unit test added

File: test/postgresql.test.js [hunk 2]
```diff
@@ -90,6 +90,21 @@ describe('postgresql connector', function() {
       });
   });
 
+  it('should reserve property `count` after query execution', function(done) {
```
Comment (bajtos) [inline test/postgresql.test.js]: `preserve`

File: test/postgresql.test.js [hunk 3]
```diff
@@ -90,6 +90,21 @@ describe('postgresql connector', function() {
...
+    Post.create(
+      {title: 'T10', content: 'C10'},
+      function(err, p) {
+        should.not.exists(err);
```
Comment (bajtos) [inline test/postgresql.test.js]: `if (err) return done(err);`

**Pull request merged**